tests/conftest.py

```diff
@@ -2,16 +2,30 @@
 from pathlib import Path
 
 import pytest
+import confuse
```

Comment:
please move all configuration related things to a `config.py`

Reply:
ok